website/src/pages/webinars/code-insights-turning-your-metrics-into-action.tsx

```diff
@@ -0,0 +1,81 @@
+        meta={{
+            title: 'Sourcegraph Code Insights: Turning Metrics into Action',
+            description:
+                'Join Sourcegraph Product Manager Joel Kwartler and Customer Engineer Shawn King to learn how to use Code Insights to turn your most pressing analytics questions into KPIs you can track with mere seconds of setup.',
+        }}
+        className="navbar-white"
+    >
+        <WebinarLayout
+            title="How to track what really matters to you and your team"
+            subtitle="Sourcegraph Code Insights: Turning Metrics into Action"
+            speakers={speakers}
+            form={{
+                formId: '66361163-5e08-4be3-8ab0-6590b70df69e',
+                onFormSubmitted: () => window.open('https://my.demio.com/recording/7BYqL99g'),
+            }}
+            description={
+                <section className="col-md-6 col-12">
+                    <p>
+                        Learn how to use Code Insights to turn your most pressing analytics questions into KPIs you can
+                        track with mere seconds of setup.
+                    </p>
+                    <p>
+                        Code Insights lets you transform your code into a queryable database and create insights in
+                        seconds for anything you can search for. You can track: migration progress, terraform versions,
+                        component reuse, code smells, vulnerability indicators, language usage, CI connections,
+                        documentation, ownership signals, and many more metrics found in your code.
+                    </p>
+                    <p>
+                        But when you can quantifiably track everything in your codebase, from migrations to code smells,
+                        how do you decide where to start?
+                    </p>
+                    <p>
+                        In this interactive session, Sourcegraph Product Manager Joel Kwartler and Customer Engineer
+                        Shawn King will teach you:
+                    </p>
+                    <ul>
+                        <li className="mb-2">What Code Insights is and why it's important</li>
+                        <li className="mb-2">
+                            Best practices for getting started from top-tier engineering organizations
+                        </li>
+                        <li className="mb-2">How to dive deeper into your data with advanced examples, including:</li>
+                        <ul>
+                            <li className="mb-2">Metadata/config tracking</li>
+                            <li className="mb-2">Commit signals</li>
+                            <li className="mb-2">Making use of advanced Sourcegraph filters</li>
+                            <li className="mb-2">
+                                Pattern tracking versions and licenses with regular expression capture groups
+                            </li>
```

Comment:
It looks like the open bullet differs from the closed bullet shown in the Figma. However, I would keep this as it is because it distinguishes these items as nested information.

Reply:
Thanks Tim! I noticed this too. We'll see what QA thinks